tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py

```diff
@@ -264,6 +264,8 @@ def get_conversion_attributes(config, dataset_size):
     dataset_name = config.get('name', '')
     conversion_parameters = copy.deepcopy(config.get('annotation_conversion', {}))
     for key, value in config.get('annotation_conversion', {}).items():
+        if config.get('command_line_mapping') and key in config['command_line_mapping'].keys():
+            value = value.relative_to(config['command_line_mapping'][key])
         if isinstance(value, Path):
             conversion_parameters[key] = str(value)
```

Comment:
I think, you need to merge these conditions, because the first expects that valie is path

Reply:
Done